### ISSUE TITLE
Opaque ffmpeg exception thrown running video colorization

### ISSUE THREAD

**Issue by covercash2** [jantic/DeOldify#310]
i was just trying to run a quick test, and ffmpeg is having an issue. the error it throws is opaque and unhelpful:

```
Error                                     Traceback (most recent call last)

<ipython-input-30-6f2a324612f0> in <module>()
      4 
      5 if source_url is not None and source_url !='':
----> 6   video_path = colorizer.colorize_from_url(source_url, 'video.mp4', render_factor, watermarked=watermarked)
      7   show_video_in_notebook(video_path)
      8 else:

3 frames

/usr/local/lib/python3.6/dist-packages/ffmpeg/_run.py in run(stream_spec, cmd, capture_stdout, capture_stderr, input, quiet, overwrite_output)
    303     retcode = process.poll()
    304     if retcode:
--> 305         raise Error('ffmpeg', out, err)
    306     return out, err
    307 

Error: ffmpeg error (see stderr output for detail)
```

the unhelpful error seems to be coming from ffmpeg.

i tried to add `capture_stderr=True` to the `run` call in `_extract_raw_frames`, but it didn't change the output.

attempting to inspect the exception doesn't produce much more information:
``` python
source_url = 'https://youtu.be/J7GY1Xg6X20' #@param {type:"string"}
render_factor = 21  #@param {type: "slider", min: 5, max: 40}
watermarked = True #@param {type:"boolean"}
if source_url is not None and source_url !='':
  try:
    video_path = colorizer.colorize_from_url(source_url, 'video.mp4', render_factor, watermarked=watermarked)
    show_video_in_notebook(video_path)
  except Exception as e:
    print(f"Unable to colorize video: {e}")
    print(f"{e.args}")
else:
    print('Provide a video url and try again.')
```
```
Unable to colorize video: ffmpeg error (see stderr output for detail)
('ffmpeg error (see stderr output for detail)',)
```

any leads would be greatly appreciated!

**Comment by jantic** [comment]
I took a quick look. Good news is that it's not all videos on YouTube that are affected.  Bad news is that it's not the usual suspects that I'd look for in diagnosing issues, and I don't have the time currently to dig deep- lots of other work going on here currently at DeOldify in our business efforts. It does appear though that this is an ffmpeg issue that needs to be addressed on their end. I'll keep this issue open for now in the meantime.

**Comment by jqueguiner** [comment]
Hi @covercash2 would it be possible to have the URL used to reproduce the error ?

**Comment by covercash2** [comment]
this is the only one I tried that didn't work: https://youtu.be/J7GY1Xg6X20

**Comment by renman87** [comment]
I had same issue; the addresses are https://www.youtube.com/watch?v=Wk4Eq8IcQMk & https://www.youtube.com/watch?v=iXT2E9Ccc8A  Both are old BW 50s Tv show, same file format, size etc. neither worked tried various verisons of the Deoldify Colab, as well as looked into the logs and googled what I could but came up with nothing. I tested a few other videos on youtube using same colab and it worked fine, so I suspect it is the source. I also tested the same url (above) as the OP, and got the same no-go response. 
I know this thread is  year old, but I wanted to at least mention it to maybe get a bugfix or at a better understanding of why some videos work and others don't.
